test/scrabble_test.rb

```diff
@@ -5,9 +5,46 @@
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
+
+  def test_it_exists
+    game = Scrabble.new
+
+    assert_instance_of Scrabble, game
+  end
+
   def test_it_can_score_a_single_letter
-    skip
-    assert_equal 1, Scrabble.new.score("a")
-    assert_equal 4, Scrabble.new.score("f")
+    game = Scrabble.new
+
+    assert_equal 1, game.score("a")
+    assert_equal 4, game.score("f")
+    refute_equal 5, game.score("f")
+  end
+
+  def test_it_can_score_a_whole_word
+    game = Scrabble.new
+
+    assert_equal 8, game.score("hello")
+  end
+
+  def test_it_can_score_empty_string_as_zero
+    game = Scrabble.new
+
+    assert_equal 0, game.score("")
+  end
+
+  def test_it_can_score_nil_input_as_nil
+    game = Scrabble.new
+
+    assert_nil game.score(nil)
   end
+
+  def test_it_is_case_insensitive
+    game = Scrabble.new
+
+    assert_equal 1, game.score("A")
+    assert_equal 1, game.score("a")
+    assert_equal 4, game.score("F")
+    assert_equal 4, game.score("f")
+  end
```

Comment:
Per comment above, breaking your `score` method into smaller methods would allow you to have some unit tests here as well.